### PR TITLE
Do not mark vsftp formula as an optional one

### PR DESCRIPTION
**Pull request by aaannz**
Due to upcoming changes in code, vsftp will no longer be optional part.
Explicitely state that for working retail config, all formulas must be
properly configured.

### CODE REVIEW

**Comment by Bischoff** [comment]
That's probably not only for doc, but also for QA, as this might make the tests fail.

Ping @srbarrios @lkotek for awareness.

File: adoc/retail_chap_formulas.adoc [hunk 1]
```diff
@@ -8,6 +8,8 @@ Formulas are pre-written Salt states, that are used to configure your {smr} inst
 
 This section lists the primary formulas shipped with {smr} and their configuration options.
 
+For working {smr} solution, all here presented formulas need to be properly configured. If you are not sure about correct configuration, use [command]``retail_branch_init`` command first. This command will create recommended formulas configuration which then can be manually edited.
```
Comment (Loquacity) [inline adoc/retail_chap_formulas.adoc]: Try this:

`All the formulas in this section must be accurately configured for your {smr} installation to function correctly.
If you are unsure of the correct formula configuration details, run the [command]``retail_branch_init`` command before you begin to create the recommended formula configuration.
You can then manually edit the formulas as required.`

File: adoc/retail_chap_formulas.adoc [hunk 2]
```diff
@@ -219,13 +221,16 @@ Use only alphanumeric characters for the branch identifier.
 [[retail.sect.formulas.saltboot]]
 == Saltboot Formula
 
-The Saltboot formula is used to configure disk images and partitioning.
-
+The Saltboot formula is used to configure disk images and partitioning for selected hardware type.
```
Comment (Loquacity) [inline adoc/retail_chap_formulas.adoc]: There's a missing article here somewhere. Possibly: "...partitioning for **the** selected hardware type."?

File: adoc/retail_chap_formulas.adoc (hunk 1 above)
Comment (aaannz) [reply adoc/retail_chap_formulas.adoc]: Sure, updated.

**Pull request merged**